source/release-notes.txt

```diff
@@ -10,6 +10,38 @@ Release Notes
    :depth: 1
    :class: twocols
 
+
+|compass| 1.43.1
+----------------
+
+*Released Jun 12, 2024*
+
+New Features:
+
+- Added a disabled state Generative AI query bar, used while Generative 
+  AI is fetching results :issue:`COMPASS-7902`.
+- Updated the close window hotkey to be ``cmd`` + ``shift`` + ``w`` to 
```

Comment:
Nonblocking Suggestion: Snooty supports a `:kbd:` role that's used for keystrokes.  I haven't used it before so I'm not sure how it looks or how widely we employ it.

Reply:
Thanks will apply to the edit, good suggestion @kennethdyer